src/services/doppler-contact-policy-api-client.ts

```diff
@@ -19,13 +19,21 @@ export interface AccountSettings {
   emailsAmountByInterval: number | null;
   intervalInDays: number | null;
   excludedSubscribersLists: SubscriberList[] | null;
+  timeRestriction: TimeRestriction | null;
 }
 
 export interface SubscriberList {
   id: number;
   name: string;
 }
 
+export interface TimeRestriction {
+  timeSlotEnabled: boolean | null;
+  hourFrom: number | null;
+  hourTo: number | null;
+  weekdaysEnabled: boolean | null;
+}
```

Comment:
Another more restrictive option could be

```suggestion
export interface TimeRestriction {
  timeSlotEnabled: false;
  hourFrom: number | null;
  hourTo: number | null;
  weekdaysEnabled: boolean | null;
} | {
  timeSlotEnabled: true;
  hourFrom: number;
  hourTo: number;
  weekdaysEnabled: boolean;
}
```